scripts/release/brew-cask.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -eq 0 ]; then
+	echo "usage: $0 <homebrew_dir> <release_version>"
+	exit
+fi
```

Comment:
@anjannath We need to add an usage function which will just print the usage if correct number of arguments are not passed  otherwise it results in to below
``` 
$ ./scripts/release/brew-cask.sh --help
./scripts/release/brew-cask.sh: line 25: cd: --: invalid option
cd: usage: cd [-L|-P] [dir]
Switched to a new branch 'minishift-'
version not found on github

$ git branch
 master
* minishift-
```